f8a_worker/workers/repo_dependency_finder.py

```diff
@@ -0,0 +1,160 @@
+"""Output: List of direct and indirect dependencies."""
+
+import requests
+import traceback
+
+from f8a_worker.base import BaseTask
+from f8a_worker.errors import TaskError
+from f8a_worker.utils import GREMLIN_SERVER_URL_REST
+from f8a_worker.workers.mercator import MercatorTask
+from f8a_worker.workers.dependency_parser import GithubDependencyTreeTask
+
+
+class RepoDependencyFinderTask(BaseTask):
+    """Finds out direct and indirect dependencies from a given github repository."""
+
+    _mercator = MercatorTask.create_test_instance(task_name='RepoDependencyFinderTask')
+
+    def execute(self, arguments=None):
+        """Task code.
+
+        :param arguments: dictionary with task arguments
+        :return: {}, results
+        """
+        self._strict_assert(arguments.get('service_token'))
+
+        github_repo = arguments.get('github_repo')
+        dependencies = []
+        repo_cves = []
+        if arguments.get('epv_list') in (None, []):
+            # self._strict_assert(arguments.get('epv_list'))
+            if len(arguments.get('epv_list')) > 0:
```

Comment:
This is redundant, right? Because if the length is zero the loop below would automatically not run.